src/admin/resolvers/mutations/ScheduledItem.ts

```diff
@@ -43,6 +43,13 @@ export async function deleteScheduledItem(
   // Access allowed, proceed as normal from this point on.
   const scheduledItem = await dbDeleteScheduledItem(context.db, data);
 
+  // Before we send the event to Snowplow, update the `updatedBy` and `updatedAt` fields
+  // as the object returned from the database resolver will have the details
+  // of the previous update and not the final one (aka the hard delete).
+  scheduledItem.updatedBy = context.authenticatedUser.username;
+  // Do we need the date in UTC here?
```

Comment:
the snowplow schema seems to call for a UTC unix timestamp, so i think we do need to [convert this](https://stackoverflow.com/a/11893157).

Reply:
@jpetto, I've double-checked here and updated the comment to reflect my findings - we do convert the date to a Unix timestamp in the `*SnowplowHandler` classes, so there is no need to do it there. Prisma also returns a JS `Date` object so it's more consistent if we set it to just `Date()` here.